profiles/templates/profiles/profile.html

```diff
@@ -3,7 +3,40 @@
 
 {% block content %}
 
-    <!-- TO DO : build profile page -->
-    TEST PROFILE PAGE
+<!DOCTYPE html>
+    <h1>Profile Page</h1>
+    <h2>You are logged in as:</h2>
 
+<!-- Profile display -->  
+ 
+    <div class="container-fluid">
+        <div class="row">
+
+            <div class="col-md-4">
+                <h4>Username :</h4>
+                <p> {{ user.username }} </p>
+            </div>
+
+            <div class="col-md-4">
+                <h4>User e-mail address :</h4>

```

Comment:
`<h4>Email Address:</h4>`
Short and sweet - just Email Address is fine.

Reply:
Changed that